### ISSUE TITLE
Shared storage is not supported by OpenShift Dedicated

### ISSUE THREAD

**Issue by cutwater** [ansible/galaxy#1502]
RWX volumes are not currently supported on OpenShift Dedicated (https://docs.openshift.com/dedicated/architecture/additional_concepts/storage.html#pv-restrictions). 

Workaround:

For development environment put all containers that require access to shared storage into single pod.

**Comment by bmbouter** [comment]
The shared storage requirement is primarily for hosting the /var/lib/pulp/ which contains Artifacts mostly. If those are hosted on S3 then maybe it won't need a RWX volume.

I ran into the same issue when I deployed Pulp3 generically onto OpenShift dedicated myself w/ these templates:  https://github.com/bmbouter/pulp3-openshift/

I used the same workaround also, placing them all on 1 pod. This kind of breaks the scaling model of OpenShift though, so we should only use it temporarily.

I opened an Issue in Pulp to test deploying Pulp without a RWX filessystem:  https://pulp.plan.io/issues/4380
